README.md

```diff
@@ -0,0 +1,93 @@
+# Conduit Connector ActiveMQ Classic
+
+The ActiveMQ Classic connector is one of [Conduit](https://conduit.io) plugins. The connector provides both a source and a destination connector for [ActiveMQ Classic](https://activemq.apache.org/components/classic/).
+
+It uses the [stomp protocol](https://stomp.github.io/) to connect to ActiveMQ.
+
+## What data does the OpenCDC record consist of?
+
+| Field                   | Description                                                                 |
+|-------------------------|-----------------------------------------------------------------------------|
+| `record.Position`       | json object with the queue name and the messageId frame header.             |
+| `record.Operation`      | currently fixed as "create".                                                |
+| `record.Metadata`       | a string to string map, with keys prefixed as `header-{STOMP_HEADER_NAME}`. |
```

Comment:
Nitpick: we should update the docs to mention that the prefix is `activemq.header.`